### PR TITLE
Bump aks operator and crd versions

### PR DESCRIPTION
**Pull request by annablender**
Issue: https://github.com/rancher/rancher/issues/36425

Bump AKS operator and crd versions to [v1.0.4-rc3](https://github.com/rancher/aks-operator/releases/tag/v1.0.4-rc3) to get oms agent panic fix.

### CODE REVIEW

File: release.yaml [hunk 1]
```diff
@@ -59,6 +59,7 @@ rancher-eks-operator-crd:
 - 100.0.2+up1.1.3-rc2
 rancher-aks-operator:
 - 100.0.3+up1.0.4-rc1
```
Comment (paynejacob) [inline release.yaml:61]: why is rc1 still in here?

**Review by paynejacob — CHANGES_REQUESTED** [review]
This pr should be in 2 commits, the first removes the old version and updates the package.  The second runs make charts.

File: release.yaml (hunk 1 above)
Comment (thedadams) [reply release.yaml:61]: It is correct for the old RC versions to be in the release.yaml file.

Comment (annablender) [reply release.yaml:61]: @thedadams Ok thanks!

**Review by thedadams — APPROVED** [review]

**Review by paynejacob — APPROVED** [review]

**Pull request merged**